### ISSUE TITLE
Changes are not reflected in compiled app if building without clean

### ISSUE THREAD

**Issue by galvanopus** [excelsior-oss/excelsior-jet-maven-plugin#11]
Create sample maven project:

```
mvn archetype:generate -DarchetypeGroupId=org.apache.maven.archetypes -DarchetypeArtifactId=maven-archetype-quickstart -DgroupId=com.example.excelsior -DartifactId=sample-application
```

Add main class reference and excelsior-jet to pom.xml:
```
<project xmlns="http://maven.apache.org/POM/4.0.0"
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
	<modelVersion>4.0.0</modelVersion>
	<groupId>com.example.excelsior</groupId>
	<artifactId>sample-application</artifactId>
	<version>1.0-SNAPSHOT</version>
	<packaging>jar</packaging>
	<name>sample-application</name>
	<url>http://maven.apache.org</url>
	<properties>
		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
	</properties>
	<dependencies>
		<dependency>
			<groupId>junit</groupId>
			<artifactId>junit</artifactId>
			<version>3.8.1</version>
			<scope>test</scope>
		</dependency>
	</dependencies>
	<build>
		<plugins>
			<plugin>
				<groupId>org.apache.maven.plugins</groupId>
				<artifactId>maven-jar-plugin</artifactId>
				<version>2.6</version>
				<configuration>
					<archive>
						<manifest>
							<mainClass>com.example.excelsior.App</mainClass>
						</manifest>
					</archive>
				</configuration>
			</plugin>
			<plugin>
				<groupId>com.excelsiorjet</groupId>
				<artifactId>excelsior-jet-maven-plugin</artifactId>
				<version>0.3.0</version>
				<configuration>
					<mainClass>com.example.excelsior.App</mainClass>
				</configuration>
			</plugin>
		</plugins>
	</build>
</project>
```

Build and launch:
```
mvn jet:build
...
>java -jar target\sample-application-1.0-SNAPSHOT.jar & target\jet\app\App.exe
Hello World!
Hello World!
```

Change `Hello World!` => `Hello World!!!`, build without clean and launch:
```
mvn jet:build
...
>java -jar target\sample-application-1.0-SNAPSHOT.jar & target\jet\app\App.exe
Hello World!!!
Hello World!
```

JAR is updated, but App.exe not.

**Comment by pjBooms** [comment]
Fixed in 0.3.2 release. Please confirm

**Comment by galvanopus** [comment]
Confirmed! Thank you.
